src/hazelweb/layout/DHDoc_Exp.re

```diff
@@ -188,6 +189,19 @@ let rec mk =
         let (doc1, doc2) =
           mk_left_associative_operands(DHDoc_common.precedence_Ap, d1, d2);
         DHDoc_common.mk_Ap(mk_cast(doc1), mk_cast(doc2));
+      | ApBuiltin(x, args) =>
+        switch (args) {
+        // TODO: How to handle tl?
+        | [hd, ..._] =>
```

Comment:
arguments are missing here -- best way to handle it is probably to call into the Ap case